modules/nw-multinetwork-sriov.adoc

```diff
@@ -40,247 +60,379 @@ and device ID 0x1015
 * Mellanox MT27800 Family [ConnectX-5] 100G card with vendor ID 0x15b3
 and device ID 0x1017
 
+== Install SR-IOV Network Operator
+
 [NOTE]
 ====
-For Mellanox cards, ensure that SR-IOV is enabled in the firmware before
-provisioning VFs on the host.
+User can choose to install the SR-IOV Network Operator by the CLI or 
+by the web console. Only one instance of the SR-IOV Network Operator is allowed
+to be deployed in a cluster
+Following the directions below to install by the CLI.
 ====
 
-== Creating SR-IOV plug-ins and daemonsets
+.Procedure
 
+. Create Namespaces for the SR-IOV Network Operator
++
 [NOTE]
 ====
-The creation of SR-IOV VFs is not handled by the SR-IOV device plug-in and
-SR-IOV CNI.
-To provision SR-IOV VF on hosts, you must configure it manually.
+You can also create the Namespaces in the web console using the *Administration*
+ -> *Namespaces* page.
 ====
 
-To use the SR-IOV network device plug-in and SR-IOV CNI plug-in, run both
-plug-ins in daemon mode on each node in your cluster.
-
-. Create a YAML file for the `openshift-sriov` namespace with the following
-contents:
+.. Create Namespace `openshift-sriov-network-operator` for the SR-IOV Network
+Operator (for example, `sriov-namespace.yaml`):
 +
-[source,yaml]
 ----
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: openshift-sriov
+  name: openshift-sriov-network-operator
```

Comment:
This namespace name depends on https://github.com/openshift/sriov-network-operator/pull/70

Reply:
change namespace back to 'sriov-network-operator' for 4.2. https://bugzilla.redhat.com/show_bug.cgi?id=1746222